coremltools/converters/tensorflow/test/test_tf_2x.py

```diff
@@ -497,6 +505,27 @@ def test_batch_norm_node_fusion(self):
                 num_batch_norm += 1
         self.assertEqual(num_batch_norm, 1)
 
+    def test_conv_bias_fusion(self):
+        x = tf.keras.layers.Input(shape=[32, 32, 3], batch_size=1)
+        conv = tf.keras.layers.Conv2D(filters=3, kernel_size=1)(x)
+        conv = tf.keras.layers.DepthwiseConv2D(kernel_size=1)(conv)
+        keras_model = tf.keras.Model(x, conv)
+        input_name = keras_model.inputs[0].name.split(':')[0]
+        model = self._test_model(keras_model=keras_model,
+                                 model_path=self.model_path,
+                                 decimal=3,
+                                 inputs={input_name: (1, 32, 32, 3)})
+        add_broadcastables = 0
+        load_constants = 0
+        for layer in model.get_spec().neuralNetwork.layers:
+            if layer.WhichOneof('layer') == 'addBroadcastable':
+                add_broadcastables += 1
+            if layer.WhichOneof('layer') == 'loadConstantND':
+                load_constants += 1
+
+        assert not add_broadcastables
```

Comment:
Just to follow the patter in the code already, can you use the unit test's `self.assertTrue` methods. We should plan on moving things to pytest style asserts but we can do that in one swoop. For now, it's better to be consistent with what's there!

Reply:
sounds good!

Also, went through and changed the other existing `assert`s to up the consistency (at least in this file)